package.json

```diff
@@ -89,9 +93,10 @@
     "react-select": "^1.0.0-beta8",
     "react-tap-event-plugin": "^0.2.1",
     "relay": "^0.8.0",
-    "request-promise": "^1.0.1",
+    "request-promise": "^2.0.1",
     "retry": "^0.8.0",
     "rollbar": "^0.5.11",
+    "snyk": "^1.11.3",
```

Comment:
Should have been, snyk must have added itself in there. It's an automated update to the deps. It think's it shouldn't there may be a reason why. I'll check the repo